browser.js

```diff
@@ -310,6 +310,11 @@ document.addEventListener('DOMContentLoaded', () => {
 	// enable OS specific styles
 	document.documentElement.classList.add(`os-${process.platform}`);
 
-	// TODO: figure out a better way to detect when React is done
-	setTimeout(init, 300);
+	// detect when React is ready before firing init
+	const awaitReact = setInterval(() => {
```

Comment:
Maybe make this reusable by making it a `waitsFor` function that accepts a selector? We need this for other things too, see the other TODO's.